gpu-aware-scheduling/docs/usage.md

```diff
@@ -65,6 +65,61 @@ Your PODs then, needs to ask for some GPU-resources. Like this:
 
 A complete example pod yaml is located in [docs/example](./example)
 
+## Node Label support
+
+GAS supports certain node labels as a means to allow telemetry based GPU selection decisions and
+descheduling of PODs using a certain GPU. You can create node labels with the
+[Telemetry Aware Scheduling](../../telemetry-aware-scheduling/README.md) labeling strategy,
+which puts them in its own namespace. In practice the supported labels need to be in the
+`telemetry.aware.scheduling.POLICYNAME/` namespace, where the POLICYNAME may be anything.
+
+The node label `gas-deschedule-pods-GPUNAME` where the GPUNAME can be e.g. card0, card1, card2...
+which corresponds to the gpu names under /dev/dri, will result in GAS labeling the PODs which
+use the named GPU with the `gpu.aware.scheduling/deschedule-pod=gpu` label. You may then
+use with a kubernetes descheduler to pick the pods for descheduling. So TAS labels the node, and
+based on the node label GAS finds and labels the PODs. Descheduler can be configured to
+deschedule the pods based on pod labels.
```

Comment:
I would remove this line from all the occurrences below and add it just before L75 in a section called Notes/Abbreviations

GPUNAME can be e.g. card0, card1, card2… which corresponds to the gpu names under /dev/dri

L76-81 I would rewrite as:

The node label `gas-deschedule-pods-GPUNAME`  will result in GAS labeling the PODs which
use the named GPU with the `gpu.aware.scheduling/deschedule-pod=gpu` label. So TAS labels the node, and based on the node label GAS finds and labels the PODs. 
You may then use a kubernetes descheduler to pick the pods for descheduling via their labels.